__tests__/views/Action.view

```diff
@@ -1,5 +1,6 @@
 Vertical
 onClick props.clickMe
+disabled props.isDisabled ? props.isDisabled : ''
```

Comment:
I notice that you're passing `disabled` to the `Vertical` which works because even though we've ignored it until your PR, `disabled` is the prop that both web and native use underneath.

However, I've been finding the convention to start boolean props with `is` to be quite handy and you're using it there too with `props.isDisabled`.

How would you feel about us calling the prop `isDisabled` instead of `disabled` in `Vertical` and `Horizontal`?

That kind of remapping would then happen [here](https://github.com/viewsdx/morph/blob/master/morph/react-dom/get-value-for-property.js). [Here's an example](https://github.com/viewsdx/morph/blob/master/morph/react-dom/get-value-for-property.js#L10-L13) in which we map `source` in an `Image` block to `src` in the `img` tag for web.

Reply:
Hi @dariocravero, 

Why do we need to map from `isDisabled` to `disabled`, won't [this line](https://github.com/amymc/morph/blob/e0a03d20d7eb9ce1f67de6cbcc2d61fe0055746b/morph/react-native/block-wrap.js#L29) cover that?